packages/auth/amplify_auth_cognito/example/integration_test/mfa_sms_totp_required_test.dart

```diff
@@ -35,7 +35,7 @@ void main() {
           signInRes.nextStep.signInStep,
           because: 'MFA is required, and TOTP is chosen when '
               'no phone number is registered',
-        ).equals(AuthSignInStep.continueSignInWithTotpSetup);
+        ).equals(AuthSignInStep.continueSignInWithMfaSetupSelection);
```

Comment:
Q: Is this expected? If so, why is it different than before?

Reply:
Yeah, it's expected because before there was only the set up totp option that users could set up when setting up MFA but now there's email added to that. So now there's an extra step injected so that users can choose which mfa method they want to set up when previously they didn't have a choice.